frontend/src/components/DropDownModal.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import styled from "styled-components";
+import { DNDSchedule } from "../models/types";
+import ExpandMoreOutlinedIcon from "@material-ui/icons/ExpandMoreOutlined";
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const TopInfo = styled.div`
+  display: flex;
+  height: 70px;
+  background-color: #fafafa;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+`;
+
+const BottomInfo = styled.div`
+  display: flex;
+  height: 200px;
+  background-color: #fafafa;
+`;
+
+export interface IDropDownModalProps {
+  schedule: DNDSchedule;
+}
+
+export interface IDropDownModalState {
+  expanded: boolean;
+}
+
+export class DropDownModal extends React.Component<
+  IDropDownModalProps,
+  IDropDownModalState
+> {
+  constructor(props: IDropDownModalProps) {
+    super(props);
+    this.state = {
+      expanded: false,
+    };
+  }
+
+  handleClick() {
+    this.setState({ expanded: !this.state.expanded });
+  }
+
+  render() {
+    return (
+      <Wrapper>
+        <TopInfo onClick={this.handleClick.bind(this)}>
+          <p>4 courses</p>
+          <p>16 credits</p>
+          <p>Freshman Standing</p>
+          <ExpandMoreOutlinedIcon></ExpandMoreOutlinedIcon>
+        </TopInfo>
+        {this.state.expanded && <BottomInfo></BottomInfo>}
```

Comment:
I think we ask users what courses they've already taken during the on-boarding process right? Do we have a way to store that info currently? Are we able to populate that information in the modal? (and also populate their schedule with those courses.)

Reply:
Not currently no, but the goal of this PR is to just get the UI in place, and we can add the functionality later once we have the onboarding process done.